File: public/phase-2-20160620022203/PYTHON/trains.py
#!/usr/bin/python
# Name : Trains
# input:
# n : number of cities
# m : number of  roads connecting the cities
# k : number of railways
# uvx : uvx corresponds to ui vi xi
# sy : uvx corresponds to si yi
elements = raw_input().strip().split(' ')
n = int(elements[0])
m = int(elements[1])
k = int(elements[2])
uvx = []
for _ in xrange(m):
	uvx_elements = raw_input().strip().split(' ')
	uvx.append(map(int, uvx_elements))
sy = []
for _ in xrange(k):
	sy_elements = raw_input().strip().split(' ')
	sy.append(map(int, sy_elements))


# write your code here
# store your results in `result`

# output

# Dummy Data
result = 2

print(result)

File: public/phase-2-20160620022203/C/trains.c
// Name : Trains
#include <stdio.h>
int main() {
	// input:
	// n : number of cities
	// m : number of  roads connecting the cities
	// k : number of railways
	// uvx : uvx corresponds to ui vi xi
	// sy : uvx corresponds to si yi
	int n, m, k, uvx[10000][3], sy[10000][2], result=2, idx, jdx;
	char uvx_tmp[3], sy_tmp[2];
	scanf("%d %d %d", &n, &m, &k);
	for(idx = 0; idx< m; idx++) {
		scanf("%s", &uvx_tmp[0]);
		for(jdx = 0;jdx<3;jdx++) {
			uvx[idx][jdx] = uvx_tmp[jdx];
		}
	}
	for(idx = 0; idx< k; idx++) {
		scanf("%s", &sy_tmp[0]);
		for(jdx = 0;jdx<2;jdx++) {
			sy[idx][jdx] = sy_tmp[jdx];
		}
	}


	// write your code here
	// store your results in `result`

	// output
	printf("%d", result);
	return 0;
}